iPSL/Electrical/Loads/PSSE/BaseClasses/baseLoad.mo

```diff
@@ -22,15 +27,15 @@ partial model baseLoad
   Real Q "Reactive power consumption (pu)";
 protected
   parameter Real anglev_rad=angle_0*pi/180;
-  parameter Real p0=S_i.re*V_0 + S_y.re*V_0^2 + S_p.re;
-  parameter Real q0=S_i.im*V_0 + S_y.im*V_0^2 + S_p.im;
+  parameter Real p0=(S_i.re*V_0 + S_y.re*V_0^2 + S_p.re)/S_b "pu";
```

Comment:
You could use `Modelica.SIunits.PerUnit` as type instead of simply `Real` whenever a type should be "pu".